Formula/uniutils.rb

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+
+class Uniutils < Formula
+  desc "Manipulate and analyze Unicode text"
+  homepage "http://billposer.org/Software/unidesc.html"
+  url "http://billposer.org/Software/Downloads/uniutils-2.27.tar.gz"
+  sha256 "c662a9215a3a67aae60510f679135d479dbddaf90f5c85a3c5bab1c89da61596"
+
+  # Allow build with clang
+  patch :DATA
```

Comment:
Has the patch been submitted upstream? Please do so, and add a comment to a commit so it can be clear when we need to remove it later.